primitives/account/src/logs.rs

```diff
@@ -136,6 +142,21 @@ pub enum Log {
 }
 
 impl Log {
+    pub fn transfer_from_transaction(transaction: &Transaction) -> Self {
```

Comment:
```suggestion
    pub fn transfer_log_from_transaction(transaction: &Transaction) -> Self {
```
I found the name a bit more confusing at first glance. Another alternative I thought of is get_transfer_log_from_transaction.

Reply:
I think your suggestion makes sense!